### PR TITLE
Add workshop meta images

### PR DESCRIPTION
**Pull request by SaraDPH**
## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/resources/azure-github-workshop/index.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 title: "Getting Started with Azure IaC and Review Stacks"
 title_tag: "Getting Started with Azure IaC and Review Stacks (July 2023 Workshop)"
 meta_desc: "In this workshop, we’ll explore the new Azure Native v2 provider and Pulumi Review Stacks on GitHub"
+meta_image: "/images/resources/Azure-IaC-Review-Stacks.png"
```
Comment (zchase) [inline themes/default/content/resources/azure-github-workshop/index.md:6]: I'd personally prefer that these images be stored in the directory as the content itself (`index.md` file). That we don't end up with long list of images in the `/resources` directory and it is clear which image maps to which workshop.

**Review by zchase — CHANGES_REQUESTED** [review]

File: themes/default/content/resources/azure-github-workshop/index.md (hunk 1 above)
Comment (SaraDPH) [reply themes/default/content/resources/azure-github-workshop/index.md:6]: > I'd personally prefer that these images be stored in the directory as the content itself (`index.md` file). That we don't end up with long list of images in the `/resources` directory and it is clear which image maps to which workshop.

I think @cnunciato said it couldn't... I'll do whatever y'all think is best, my goal is just to get a meta image, where it should go it's up to you. :)

Comment (SaraDPH) [reply themes/default/content/resources/azure-github-workshop/index.md:6]: Why is the Kubernetes workshop image not showing?!

**Review by sean1588 — APPROVED** [review]

File: themes/default/content/resources/azure-github-workshop/index.md (hunk 1 above)
Comment (cnunciato) [reply themes/default/content/resources/azure-github-workshop/index.md]: @zchase Yep, agreed -- we tried this first, but saw that the templates weren't set up for it yet, so didn't want to block on it. But we'll come back and make that work.

**Pull request merged**